addons/chatmon/trigger_class.lua

```diff
@@ -0,0 +1,67 @@
+local trigger_class = {}
+local meta = {__index = trigger_class}
+
+local function get_match(match)
+    if match ~= '<name>' then
+        return match
+    end
+
+    -- this is done to have parity with the old plugin.
+    local player_name = windower.ffxi.get_player().name:lower()
+    return '* ' .. player_name .. '|'
```

Comment:
I still say we replace this with a simple regex match with `'\\b' .. player_name .. '\\b'`.

Reply:
I honestly am not sure it is necessary to begin with, i think it could just be `*player_name*`

Reply:
The only problem with just using `player_name` directly, is that it would match it if it was part of a string. So if someone was named `Bob` it would trigger on `I bobbed for apples yesterday`. Hence regex with `\\b` around the player name for boundary matching seems most reasonable.